### PR TITLE
Add Homework 5

### PR DESCRIPTION
**Pull request by eogenio777**

### CODE REVIEW

File: src/main/java/com/epam/tc/hw3/page/objects/composite/component/TopMenuComponent.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.epam.tc.hw3.page.objects.composite.component;
 
+import io.qameta.allure.Step;
```
Comment (akulebyakin) [inline src/main/java/com/epam/tc/hw3/page/objects/composite/component/TopMenuComponent.java]: unused import

File: src/test/java/com/epam/tc/hw5/cucumber/hook/CucumberHooks.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    private WebDriver driver;
+
+    @BeforeAll
+    public static void setUpAll() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @Before
+    public void setUp() {
+        driver = new ChromeDriver();
```
Comment (akulebyakin) [inline src/test/java/com/epam/tc/hw5/cucumber/hook/CucumberHooks.java:22]: Please set up driver window to maximized size as it's one of requirements

**Review by akulebyakin — CHANGES_REQUESTED** [review]
Please review comments

**Review by akulebyakin — APPROVED** [review]
Approved. Good job

File: src/main/java/com/epam/tc/hw3/page/objects/composite/component/TopMenuComponent.java (hunk 1 above)
Comment (eogenio777) [reply src/main/java/com/epam/tc/hw3/page/objects/composite/component/TopMenuComponent.java]: Import removed

File: src/test/java/com/epam/tc/hw5/cucumber/hook/CucumberHooks.java (hunk 2 above)
Comment (eogenio777) [reply src/test/java/com/epam/tc/hw5/cucumber/hook/CucumberHooks.java:22]: Window maximized

**Pull request merged**